okio/jvm/jmh/build.gradle

```diff
@@ -23,10 +23,10 @@ jmh {
 }
 
 dependencies {
-  compile project(path: ':okio', configuration: 'runtimeElements')
+  compile project(path: ':okio', configuration: 'jvmRuntimeElements')
```

Comment:
Is this needed? I thought you changed all these to not specify the configuration explicitly

Reply:
Nope, it's not needed here, fixed